#311 [문서 페이지] 모바일에서 상단 메뉴 정렬 수정

smaker opened the pull request:
## 이슈 링크
> #311

## 오류 원인
> 화면 너비와 상관없이, 메뉴 부분을 가로로 쭉 나열함

## 해결 방안
> viewport가 768px 이하인 경우, 해당 메뉴 영역의 정렬을 변경함

smaker committed:
enhancement: #304 Select 컴포넌트에 className을 넘길 수 있도록 개선

smaker committed:
enhancement: #304 [문서 페이지] 모바일에서의 상단 메뉴 부분 정렬 수정

devgoodway commented:
1. PC 화면에서 가로 사이즈 고정되는 문제 : 너비에 따라 조절되도록 해주세요.
![image](https://github.com/bmrdevteam/Altsis/assets/22511259/2b0bc9ae-7943-4f5a-9884-a7f466836338)

2. 메뉴가 문서 화면과 겹치는 문제 해주세요.
![image](https://github.com/bmrdevteam/Altsis/assets/22511259/8e40175f-5bcf-4c61-b378-5c2c562d3643)

smaker committed:
fix: #311 [문서 페이지] 상단 메뉴 스타일 조절
- PC 화면에서 상단 메뉴 부분의 각 컬럼의 너비가 가변적으로 변경되도록 수정
- 모바일 화면에서 상단 메뉴와 컨텐츠 영역이 겹치는 문제 수정
